prac_05/emails.py

```diff
@@ -0,0 +1,30 @@
+# Time estimate: 30 minutes
+
```

Comment:
If you have a function, its better to also have a main function and call the main in the end of the program